cmd/virt-handler/virt_launcher.cil

```diff
@@ -6,5 +6,8 @@
     (allow process self (tun_socket (relabelfrom relabelto attach_queue)))
     (allow process hugetlbfs_t (dir (add_name create write remove_name rmdir setattr)))
     (allow process hugetlbfs_t (file (create unlink)))
+    (allow process nfs_t (dir (mounton)))
```

Comment:
just double checking. Does the fact we're using memfd change anything with selinux? are these entries still required?

Reply:
It changes. However, since we've decided to have a way to fall back to the previous hugepages allocation method (without memfd) - we need to keep this as well.
We could